.gitattributes

```
@@ -0,0 +1,2 @@
+*.rs    diff=rust
+*.md    diff=markdown
```

```suggestion
*.rs diff=rust
*.md diff=markdown
```
Are these indents necessary?

---

No, I just used tab indent :slightly_smiling_face: